test/remote/gateways/remote_pay_trace_test.rb

```diff
@@ -269,15 +277,15 @@ def test_refund_without_amount
     settle = @gateway.settle()
     assert_success settle
 
-    refund = @gateway.refund(authorization)
+    refund = @gateway.refund('', authorization)
     assert_success refund
     assert_equal 'Your transaction was successfully refunded.', refund.message
   end
 
   def test_failed_refund
-    response = @gateway.refund('', @options)
+    response = @gateway.refund(100, '', @options)
     assert_failure response
-    assert_equal 'One or more errors has occurred.', response.message
+    assert_equal 'Errors- code:981, message:["Log in failed for insufficient permissions."]', response.message
```

Comment:
:squint: this seems like an unexpected error message in this context?

Reply:
Per our chat, this has been revised!